src/plugins/index_pattern_management/opensearch_dashboards.json

```diff
@@ -4,5 +4,5 @@
   "server": true,
   "ui": true,
   "requiredPlugins": ["management", "data", "urlForwarding"],
-  "requiredBundles": ["opensearchDashboardsReact", "opensearchDashboardsUtils"]
+  "requiredBundles": ["opensearchDashboardsReact", "opensearchDashboardsUtils", "savedObjects"]
```

Comment:
savedObjects is a core service right?

Reply:
should be the plugin savedObjects https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/saved_objects